### PR TITLE
Add UTs for covering libnetwork

### PR DESCRIPTION
**Pull request by robbrockbank**
Added more UTs for libnetwork.

I have currently added pragmas to not count the veth stuff.  I'll sort that out when we move it to netns.

### CODE REVIEW

**Comment by robbrockbank** [comment]
I've added a few more TODOs in here... something to look at later.

File: calico_containers/tests/unit/docker_plugin_test.py [hunk 1]
```diff
@@ -15,16 +15,21 @@
...
 from pycalico.datastore_datatypes import Endpoint
+from pycalico.datastore_errors import DataStoreError
 
 TEST_ENDPOINT_ID = "TEST_ENDPOINT_ID"
 TEST_NETWORK_ID = "TEST_NETWORK_ID"
 
+# Expected 500 error response.
+ERROR_RESPONSE_500 = {"Err": "500: Internal Server Error"}
+
 #@TODO Misuse of Mock - when mocking out a function, safest to use patch.
```
Comment (tomdee) [inline calico_containers/tests/unit/docker_plugin_test.py]: Is this resolved now?

Comment (robbrockbank) [inline calico_containers/tests/unit/docker_plugin_test.py]: ah yes!  i'll remove.

**Comment by tomdee** [comment]
LGTM. Lets talk about the TODOs next week.

**Pull request merged**